### PR TITLE
Navigation added in CL

### PR DESCRIPTION
**Pull request by humzahasan**

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
The code looks good to me. Good to merge.

**Review by tonymike99 — APPROVED** [review]
Nice code. Good to merge.

File: library/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.nav-link {
+  margin: 0.5rem;
```
Comment (iRohitGaur) [inline library/navbar/navbar.css:32]: suggestion: give this value in px if it will not change

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**